### PR TITLE
Key list templating

### PR DESCRIPTION
**Pull request by rmnbrd**

### CODE REVIEW

File: src/components/KeyList/index.js [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
+}
+
+const KeyList = ({ keys }: Props) => {
   return (
     <ListWrapper>
-      <KeyItem isEven />
-      <KeyItem isEven={false} />
-      <KeyItem isEven />
-      <KeyItem isEven={false} />
+      {keys.map(k => <KeyItem key={k.key} value={k} isEven />)}
```
Comment (Tom-Bonnike) [inline src/components/KeyList/index.js]: `isEven={index % 2 === 0}`

File: src/components/ProjectMenu/index.js [hunk 2]
```diff
@@ -17,7 +17,11 @@ type Props = { router: any, href: string }
 const ProjectMenu = ({ router, href }: Props) => (
   <Wrapper mTop="xlarge">
     {MENU_ITEMS.map(({ route, icon, label }) => (
-      <Link route={route} params={{ projectSlug: router.query.projectSlug }}>
+      <Link
+        key={route}
```
Comment (Tom-Bonnike) [inline src/components/ProjectMenu/index.js]: Ah oops, thanks

File: src/containers/SingleProjectPageContainer/index.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ type Props = {
 const SingleProjectPageContainer = ({ projectSlug }: Props) => (
   <Wrapper flex contentCentered stretch>
     <Query query={query} variables={{ projectSlug }}>
-      {({ error, loading, data }) => {
+      {({ error, loading, data }: any) => {
```
Comment (Tom-Bonnike) [inline src/containers/SingleProjectPageContainer/index.js]: 😄 This didn’t work for me 🤔

File: src/components/KeyItem/index.js [hunk 4]
```diff
@@ -38,6 +40,8 @@ class KeyItem extends Component<Props, State> {
   render() {
     const { isEven } = this.props
     const { detailsOpened } = this.state
+    const { value } = this.props
```
Comment (Tom-Bonnike) [inline src/components/KeyItem/index.js]: you should probably use line 41 for that

File: src/components/ProjectMenu/index.js (hunk 2 above)
Comment (rmnbrd) [reply src/components/ProjectMenu/index.js]: 😉

**Pull request merged**